doc/semitrans.xml

```diff
@@ -376,3 +376,19 @@ gap> Size(WW);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="SmallGeneratingSet">
+   <ManSection>
+    <Oper Name="SmallGeneratingSet" Arg="G"/>
+    <Returns>A small generating set with maximal idempotent elements.</Returns>
+    <Description>
+      When <A>G</A> is a subgroup of the Partition Monoid on <A>n</A> points. 
+	  of the form <A>N</A> union <A>SingularPartitionMonoid(n)</A> where <A>N</A>
```

Comment:
Use `<A>...</A>` for function arguments, and `<C>...</C>` for code such as `SingularPartitionMonoid(n)`.